### PR TITLE
UpsellNudge: Set forceHref to true for UpsellNudges in sidebar A/B test

### PR DESCRIPTION
**Pull request by sixhours**
#### Changes proposed in this Pull Request

* Adds `forceHref` prop to UpsellNudges in the sidebar A/B test to make the whole banner click-able
* Hides chevron on compact UpsellNudges that are linked to conserve space
* Update test date to start on Friday (tentative)
* Also cleans up styles to avoid using `&` feature in SCSS for easier search-ability

One question: 

* Wrapping a `button` or `div` element with an anchor tag is invalid XHTML. Is this is a blocker? We're already doing this in the `Banner` component when `forceHref` is used with a call to action. I'm hesitant to refactor `Banner` to avoid stepping on existing nudges' toes.

#### Testing instructions

* Switch to this PR
* Open a free site; you should see the existing JITM in the sidebar.
* Switch to the `variantShowUnifiedUpsells` for the `sidebarUpsellNudgeUnification` test
* You should be able to click on the whole upsell, not just the button
* Tracks events should also fire correctly for both the button click or the banner click (but not be duplicated)

### CODE REVIEW

File: client/lib/abtest/active-tests.js [hunk 1]
```diff
@@ -128,7 +128,7 @@ export default {
 		localeTargets: 'any',
 	},
 	sidebarUpsellNudgeUnification: {
-		datestamp: '20200127',
+		datestamp: '20200221',
```
Comment (gwwar) [inline client/lib/abtest/active-tests.js]: @martinremy is it okay if we reuse the test name here?

Comment (martinremy) [reply client/lib/abtest/active-tests.js]: Yep, you can reuse the test name with a new date, that will give us a unique slug for new Hypotheses.

**Comment by gwwar** [comment]
> Wrapping a button or div element with an anchor tag is invalid XHTML. Is this is a blocker?

It looks a little odd, but it's valid in HTML5 to let anchor tags wrap flow content.

http://w3c.github.io/html-reference/a.html#a-changes

> Although previous versions of HTML restricted the a element to only containing phrasing content (essentially, what was in previous versions referred to as “inline” content), the a element is now transparent; that is, an instance of the a element is now allowed to also contain flow content (essentially, what was in previous versions referred to as “block” content)—if the parent element of that instance of the a element is an element that is allowed to contain flow content.

http://w3c.github.io/html-reference/common-models.html#common.elem.flow

**Comment by gwwar** [comment]
@sixhours were we waiting for a specific date for this one to merge?

**Comment by sixhours** [comment]
> @sixhours were we waiting for a specific date for this one to merge?

I don't think so! I wasn't sure if we needed other approval from martin or Reader folks before I shipped, though.

**Comment by gwwar** [comment]
@sixhours You should be all set then to 🚢 unless we needed more data on our current test run. Martin didn't flag the date bump 👍

**Pull request merged**